docs/contributing/code-contributions.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Code Contributions
```

Comment:
This document covers contributing content so I find the title misleading, but I don't really have better alternative

Reply:
I can see that. The experience is pretty intertwined, I didn't see a clear way to separate them without repeating stuff....but happy to take suggestions

Reply:
Ah here's why it's confusing: blog content and the website are all grouped into "code contributions". I could make separate pages for "blog contributions" and "website contributions", but it's a slippery slope to too many pages.